content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,216 @@
+:menuselection:`Inventory --> Configuration --> Settings --> Operations`. Check the box beside
+:guilabel:`Packages`, and click :guilabel:`Save` to activate the feature.
 
-The packaging is the physical container that protects your product. If
-you are selling computers, the packaging contains the computer with the
-notice and the power plug.
+On a delivery order, assign which package to use by clicking on the :guilabel:`Detailed Operations`
+icon (represented by 4 horizontal lines) on the right of each product in the :guilabel:`Operations`
+tab.
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+.. image:: usage/detailed-operations-icon.png
+   :align: center
+   :alt: Find detailed operations icon to the right on the product line
 
-.. image:: usage/usage_07.png
-    :align: center
+Clicking the :guilabel:`Detailed Operations` icon of a product reveals a pop-up window. In this
+window, the :guilabel:`Product` name, :guilabel:`Demand`, and :guilabel:`Quantity Done` are clearly
+displayed above the customizable product line.
 
-.. image:: usage/usage_08.png
-    :align: center
+Here, the warehouse location the product is coming from can be modified, in addition to the
+:guilabel:`Source Package` and :guilabel:`Destination Package`, which represents what package(s) the
+product(s) will go into.
+
+Add a line for each additional package to be used, by clicking :guilabel:`Add a line`. Entering the
+desired amount in the :guilabel:`Done` field specifies the amount of the product to go into the
+:guilabel:`Destination Package`.
+
+.. example::
+   Although 1 delivery order is typically associated with 1 package, orders can also be split using
+   the :guilabel:`Detailed Operations` window. Package 10 boxes of pencils with 2 boxes of erasers
+   by selecting the :guilabel:`Detailed Operations` icon to the right of `Box of Pencils`.
+
+   The resulting window is used to detail how the 20 products `Box of Pencils`, are to be packaged.
+   Fill in `10` under the :guilabel:`Done` field to package 10 products into the
+   :guilabel:`Destination Package`. Start typing under this field and select the :guilabel:`Create`
+   button to create a new package, `PACK0000006`. Click :guilabel:`Add a line` to assign the
+   remaining products in another package, `PACK0000007`.
+
+   .. image:: usage/packages-detailed-14-15.png
+      :align: center
+      :alt: Detailed operations where the amount of product going in a pack can be specified.
+
+   Select :guilabel:`Detailed Operations` on the product line for `Box of Erasers` and similarly
+   set 2 :guilabel:`Done` products to `PACK0000006` and `PACK0000007` each.
+
+   .. image:: usage/packages-14-15-out.png
+      :align: center
+      :alt: Put in pack once the done amount matches the demand.
+
+   Once all the products for each line have been entered, click :guilabel:`Confirm` to return to the
+   :abbr:`DO (Delivery Order)`.
```

Comment:
@ksc-odoo Hi KC!
This is the new section I added that I wanted your suggestions on. I thought to put this section in an example admonition block because I spoke to an inventory expert from support, and they mentioned that assigning multiple packages in a delivery order isn't really a common flow. But I still wanted to show readers how to use detailed operations so I threw it into a giant "example" admonition block.

Is the block too long? Otherwise, we can just take it out. Thanks again, KC!

Reply:
I totally see what you're saying. Really great addition to the doc, by the way. I get why you think it *might* be too long for an "Example" block, but I think it could work.

Have you considered making this a sub-section of Packages? You could title it something like "Multiple packages in a delivery order"... and speak in more general terms and add this as an example...? Not sure , though. Completely your call - but I do think it's valuable information that belongs in the doc. Nicely done!

Reply:
Also, it may be worth considering using the written "one" (or "single") instead of "1" for instances like *"...a single delivery order is associated with a single package..."*  - but that's more of an opinion - not a requirement. Just something to think about...